language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/FunctionTypeDescriptorNodeContext.java

```diff
@@ -77,7 +77,7 @@ public FunctionTypeDescriptorNodeContext() {
     }
 
     private boolean withinParameterContext(BallerinaCompletionContext context, FunctionTypeDescriptorNode node) {
-        FunctionSignatureNode functionSignatureNode = node.functionSignature();
+        FunctionSignatureNode functionSignatureNode = node.functionSignature().get();
```

Comment:
shall we do the` isEmpty()` check for the optional? we can combine with the https://github.com/ballerina-platform/ballerina-lang/pull/28233/files?file-filters%5B%5D=.java&file-filters%5B%5D=.json#diff-2f74de221808a7653a95000c5953e1384d417c48e3d390754c82d521ffec281aL81

Reply:
Fixed.